python/paddle/tensor/creation.py

```diff
@@ -208,20 +208,20 @@ def to_tensor(data, dtype=None, place=None, stop_gradient=True):
             value=data,
             place=place,
             persistable=False,
-            zero_copy=True,
+            zero_copy=False,
```

Comment:
the doc:" if the data is an numpy\.ndarray of with the same ``dtype``  and the current place is cpu, no copy will be performed". should be changed too.

Reply:
done, thx